generate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# set -eu
+
+declare -A CMD=(
+	[apache]='apache2-foreground'
+	[fpm]='php-fpm'
+	[fpm-alpine]='php-fpm'
+)
+
+declare -A BASE=(
+	[apache]='debian'
+	[fpm]='debian'
+	[fpm-alpine]='alpine'
+)
+
+declare -A EXTRAS=(
+	[apache]='¬RUN a2enmod rewrite'
+	[fpm]=''
+	[fpm-alpine]=''
+)
+
+VERSION="${1:-$(curl -sS https://roundcube.net/VERSION.txt)}"
```

Comment:
```suggestion
VERSION="${1:-$(curl -fsS https://roundcube.net/VERSION.txt)}"
```
so curl fails on HTTP errors.